### ISSUE TITLE
`function-parentheses-newline-inside` and `media-feature-parentheses-space-inside` under `always` grow a character a run inside parentheses holding nothing but whitespace

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#329]
Found while fixing #272, and reproduced on `main` byte for byte, so that branch neither adds it nor touches it.

`postcss-value-parser` hands back everything a pair of parentheses holds as the `before` of the function node whenever there is no node between them, and leaves `after` empty however the file spells it: `valueParser("f( )").nodes[0]` comes back with `before` of one space and `after` of none. Both rules read the two sides apart. The opening side is answered by what `before` holds and is satisfied by the first fix; the closing side asks about `after` alone, which is empty and stays empty however much is written there, since the next run hands the whole run back as `before` again. So the closing half of `always` is reported on every run and its fix writes another character on every run.

A comment between the parentheses settles both rules at once, since that is a node, and the whitespace behind it is then `after` like any other. `function-parentheses-space-inside` is not in this: it turns away a function holding no arguments before it reads either side, so `f()` and `f( )` come back untouched under all four of its options — the option is not enforced there, but nothing runs away either.

### What minimal example or steps are needed to reproduce the bug?

```css
a { b: f(); }
a { b: f( ); }
@media () { a { b: c; } }
@media ( ) { a { b: c; } }
```

The first two for `function-parentheses-newline-inside`, the last two for `media-feature-parentheses-space-inside`. Each reproduces on its own.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": [
		"@stylistic/stylelint-plugin"
	],
	"rules": {
		"@stylistic/function-parentheses-newline-inside": "always",
		"@stylistic/media-feature-parentheses-space-inside": "always"
	}
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

One run of `--fix` leaving a file the rule has nothing more to say about, which is what both rules do for every pair of parentheses that holds a node.

### What actually happened?

A character a run, for as long as the fixer is asked, and a clean report every time:

```css
a { b: f(

); }
```

```css
@media (    ) { a { b: c; } }
```

Four runs of the first fixture and four of the third. Under `postcss-scss` and `postcss-less` the same.

### Do you have a proposal to fix the bug?

The two sides of such a pair of parentheses are one run of whitespace and not two, and the closing question has to be asked about that run rather than about the empty text the parser calls `after`. `getCheckAfter` already walks back over the nodes of the function to gather the whitespace behind the last significant one; where there is no significant node at all, the whitespace it walks back over is the very run the opening side reads, and the option is already satisfied by the break or the space written there.

The write wants the same answer: what a fix puts in front of the closing parenthesis of such a pair belongs behind whatever the opening fix wrote, and both belong in the one span the parentheses enclose.
